### ISSUE TITLE
Release a new version

### ISSUE THREAD

**Issue by AMeng** [instrumenta/kubeval#258]
There has been a lot of great stuff merged to master since 0.15.0 was released. Would it be possible to release a new version?

**Comment by John-Lin** [comment]
same here. Waiting for new features in next release.

**Comment by yutachaos** [comment]
same here. 
I'm waiting very very so much !!!

**Comment by 06kellyjac** [comment]
Would be nice if it was fixed to work correctly with go modules versioning (see the details section here https://pkg.go.dev/github.com/instrumenta/kubeval)
You might need to do a major version bump

**Comment by dcchambers** [comment]
Bumping this issue.

**Comment by carlossg** [comment]
I have cut a 0.16.0 but I don't have rights to push it to the homebrew tap @garethr 
https://github.com/instrumenta/kubeval/releases/tag/0.16.0

**Comment by dcchambers** [comment]
That's awesome @carlossg! Thank you! It looks like there's [a bot that publishes new versions to the tap](https://github.com/instrumenta/homebrew-instrumenta/commit/e31015cd7c5431adc5c2d955068daec68ab54ad7) but there isn't any documentation on how it works/is triggered...

**Comment by haines** [comment]
>Would be nice if it was fixed to work correctly with go modules versioning

I agree - isn't this just a case of tagging releases as `v0.16.0` rather than `0.16.0`? https://github.com/instrumenta/kubeval/issues/182#issuecomment-576849569

**Comment by carlossg** [comment]
I've pushed v0.16.1 https://github.com/instrumenta/kubeval/releases/tag/v0.16.1

**Comment by mycrEEpy** [comment]
Seems like the images for v0.16.x are missing in Docker Hub?

**Comment by carlossg** [comment]
I only have rights in this repo. homebrew, docker images,... I can't release

**Comment by dcchambers** [comment]
If anyone installs Kubeval locally via Homebrew, I added [a tap for it with v0.16.1](https://github.com/dcchambers/homebrew-tap/blob/master/Formula/kubeval.rb) since the [official tap](https://github.com/instrumenta/homebrew-instrumenta) hasn't been updated.

```sh
brew uninstall kubeval # Otherwise brew will return an error.
brew tap dcchambers/tap
brew install dcchambers/tap/kubeval
```

**Comment by Vivida1** [comment]
> I only have rights in this repo. homebrew, docker images,... I can't release

Who can then? We would also like to continue using kubeval in Docker in our pipelines. Right now we are stuck with version 15.0. Why not create a new Docker Hub account with instrumenta/kubeval?
